packages/scope-manager/tests/fixtures/class/declaration/index-signature.ts

```diff
@@ -0,0 +1,3 @@
+class Foo {
+  [x: string]: any;
```

Comment:
I think we should also use `[x: Foo]: string` and `[x: string]: Foo` here, otherwise the snapshot doesn't contain anything about the index signature